### ISSUE TITLE
Testnet - After cancelling retiremnet, my pools still shows as going to retire

### ISSUE THREAD

**Issue by os11k** [ADA-Ocean-Australia/adatools.io-support#11]
My pool seems to work ok and I tested it a bit, but then I decided to retire it at epoch 165. After that I found one issue with my redundancy design and decided to un-retire it and test it a bit more. Now seems adatools.io do not shows my pool at all and it is still listed there as going to retire pool. And relays are listed as Uncontactable. By opposite on pooltool.io I see my pool normally and I can still see my pool in testnet daedalus as normal pool, before it was showing as going to retire. My pool URL:

https://testnet.adatools.io/pools/e82e9f7930bcf28a15369de34d4f5c32adad5cbd0ab050f88b52ce8b

Is it a bug?

**Comment by CyberCyclone** [comment]
This is a known bug. Will have a fix out soon.

**Comment by CyberCyclone** [comment]
I've made the changes to testnet.adatools.io so that pool retirements appear again if it detects a new registration certificate with a created date after the pool retirement was submitted. This is only on testnet as I have a few things to check, but it should be operating correctly. I'll be planning to release on Sunday to mainnet.

Once this has been released to mainnet I'll close this issue. After that if you think there's still issues then feel free to open it again.
